packages/spanner/src/transaction.js

```diff
@@ -98,7 +98,7 @@ function Transaction(session, options) {
   this.retries_ = 0;
   this.runFn_ = null;
 
-  this.timeout_ = 60000;
+  this.timeout_ = 3600000;
```

Comment:
Could we just require the config file so we don't have to mange it in multiple places?

Reply:
That makes sense to me.

Reply:
I agree. Currently if config file is present does this value get overridden by the config file value somewhere?